src/components/theme/SearchWidget/SearchWidget.jsx

```diff
@@ -81,6 +81,10 @@ class SearchWidget extends Component {
     this.props.history.push(
       `/search?SearchableText=${encodeURIComponent(this.state.text)}${path}`,
     );
+    // reset input value 
```

Comment:
```suggestion
    // reset input value
```

The lint check doesn't like the extra space at the end of the line.